index.html

```diff
@@ -195,16 +267,16 @@ <h2 class="aboutme-title">About me</h2>
 				<p class="aboutme-text">Hello I'm a software developer! I can help you build a product , feature or website Look
 					through some of my work and experience! If you like what you see and have a project you need coded, don't
 					hestiate to contact me.</p>
```

Comment:
- [ ] Please make sure that this section matches the one on the Figma design, as it is missing a background at the moment like shown below 🔽 

### Figma Design
![figma about](https://user-images.githubusercontent.com/104319462/220788237-8a971327-4c52-42b0-b22e-29c9738cd144.png)

### Your Design
![your about](https://user-images.githubusercontent.com/104319462/220788247-db2f8f63-4440-4d6b-91cc-6d76397277fa.png)